### ISSUE TITLE
UNSTABLE inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench_dynamic)

### ISSUE THREAD

**Issue by izaitsevfb** [pytorch/pytorch#106898]
We have two inductor torchbench tests failing since today:

https://www.torch-ci.com/hud/pytorch/pytorch/main/1?per_page=50&name_filter=nductor%20%2F%20cuda11.8-py3.10-gcc7-sm86%20%2F%20test%20(inductor_torchbenc

inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench_dynamic, 1, 1, linux.[g5](https://internalfb.com/intern/bunny/?q=g5).4xlarge.nvidia.gpu)
inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench, 1, 1, linux.[g5](https://internalfb.com/intern/bunny/?q=g5).4xlarge.nvidia.gpu)

Error: 9 models have accuracy status regressed:
    Background_Matting LearningToPaint clip doctr_det_predictor doctr_reco_predictor drq sam soft_actor_critic yolov3

The root cause seem to be cv2 becoming incompatible, and it’s not caused by any particular commit:
```
File "/opt/conda/envs/py_3.10/lib/python3.10/site-packages/cv2/typing/__init__.py", line 69, in <module>
    NumPyArrayGeneric = numpy.ndarray[typing.Any, numpy.dtype[numpy.generic]]
TypeError: 'numpy._DTypeMeta' object is not subscriptable
```

cc @seemethere @malfet @pytorch/pytorch-dev-infra

**Comment by izaitsevfb** [comment]
Confirmed to be caused by cv2 release 7 hours ago https://pypi.org/project/opencv-python/#history

**Comment by izaitsevfb** [comment]
Fixed in #106900
